hack/upgrade-test-operator-sdk.sh

```diff
@@ -296,6 +296,20 @@ VIRTIOWIN_IMAGE_CM=$(${CMD} get cm virtio-win -n ${HCO_NAMESPACE} -o jsonpath='{
 
 [[ "${VIRTIOWIN_IMAGE_CSV}" == "${VIRTIOWIN_IMAGE_CM}" ]]
 
+Msg "check that the kubevirt console plugin is accessible"
+EXPECTED_PLUGIN_NAME="kubevirt-plugin"
+# confirm the ConsolePlugin exists in the cluster
+${CMD} get consoleplugin ${EXPECTED_PLUGIN_NAME}
+PLUGIN_SERVICE_NAME=$(${CMD} get consoleplugin ${EXPECTED_PLUGIN_NAME} -o jsonpath='{.spec.backend.service.name}')
+PLUGIN_SERVICE_PORT=$(${CMD} get consoleplugin ${EXPECTED_PLUGIN_NAME} -o jsonpath='{.spec.backend.service.port}')
+
+# get a console pod
+CONSOLE_POD=$(${CMD} get pods -n openshift-console -l app=console,component=ui -o json | jq -r '.items[0].metadata.name')
+# verify the plugin manifest file is reachable to the console pod, and extracts from it the plugin name
+PLUGIN_NAME=$(${CMD} exec -n openshift-console "${CONSOLE_POD}" -- curl -ks https://${PLUGIN_SERVICE_NAME}.${HCO_NAMESPACE}.svc:${PLUGIN_SERVICE_PORT}/plugin-manifest.json | jq -r '.name')
+# verify the plugin name in the manifest matches the expected name
+[[ "${PLUGIN_NAME}" == "${EXPECTED_PLUGIN_NAME}" ]]
+
```

Comment:
Do we still need this bash test?

Reply:
if the ginkgo functional tests are being executed anyway in the upgrade lanes, then yes,  the bash test is redundant.

Reply:
All the tests are now running in all the upgrade lanes. both bash and go.